definitions/checks/disk/performance.rb

```diff
@@ -37,16 +32,29 @@ def run
         end
       end
 
+      def default_dirs
+        @default_dirs ||= %i[pulp2 pulpcore_database mongo foreman_database].inject({}) do |dirs, f|
+          if feature(f) && File.directory?(feature(f).data_dir)
```

Comment:
FYI, currently there is no `data_dir` method for feature `pulpcore_database`.

Reply:
Hello as we discussed data_dir is in concern. The tests were failing because on 2.3.3, I pushed new changes, these are showing tests as green.

Reply:
Thanks @upadhyeammit. Not sure why job is showing pending status but tests are passed for both ruby versions.

Reply:
> Thanks @upadhyeammit. Not sure why job is showing pending status but tests are passed for both ruby versions.

yep, I just forced pushed it again, and its completed now !